Style first pass

alexelash opened the pull request:
The main changes were in CSS and adding a few images, but I made some other adjustments to the HTML:
- Added a site `<title>` for the browser tab
- Added meta description for search results (took text from this repo's intro)
- Made a few semantic adjustments (ensuring that `<ul>` only has `<li>` in it, using `<button>` element where appropriate, `<label>` for inputs vs `<legend>`, etc)
- Opinionated decision to make most text sentence case vs. lowercase

Notes:
- I'm using CSS Variables... not sure what the browser coverage is for them (I think major browsers are good)

Commit:
style

Commit:
style

Commit:
Merge branch 'master' into feature/al/style

pcaisse reviewed (commented):
This looks awesome! I love the attention to detail with the drop shadow around the legend, the borders around the color swatches, the horizontal rule, and the little 'x' not just floating out in space anymore :smile: The filter box looks similarly sweet. Links and filter toggles look good. Cursor game on point.

The only thing that we might want to change is the color of the links which seem to be white globally. This makes the links in the modal (as well as the OSM link in the bottom right of the map) hard to see:
![about_white_links](https://user-images.githubusercontent.com/2926237/49736044-06214c80-fc57-11e8-842c-daa9b40ee659.png)

One other thing I noticed is that the SVGs look weird to me at my default font size on my home machine. If I zoom in to 110%, it looks fine :man_shrugging: 

100%
![less_filters](https://user-images.githubusercontent.com/2926237/49736236-8e9fed00-fc57-11e8-91f6-4f07a111d664.png)

110%
![less_filters_zoomed](https://user-images.githubusercontent.com/2926237/49736300-be4ef500-fc57-11e8-9b30-8f9c469c9fcb.png)

Also, this is a total nitpick, but I feel like the bold font makes the percentages in the legend harder to read... Without the boldness, numbers in that font do look really skinny, so maybe a different font would be better? What do you think?

Legend (bold)
![legend_bold](https://user-images.githubusercontent.com/2926237/49736674-c491a100-fc58-11e8-9b2c-28fee6e7c882.png)

Legend (no bold)
![legend_no_bold](https://user-images.githubusercontent.com/2926237/49736685-c9eeeb80-fc58-11e8-8b4e-2782931b5a03.png)

Thanks you so much for this PR! :bowing_man:

Commit:
Merge branch 'feature/al/style' of github.com:pcaisse/language-map-us into feature/al/style

alexelash commented:
@pcaisse No prob! Ready for another look, I've:
- Fixed default links so they're no longer white
- Upped the SVG size by 10%
- Changed the font and removed bold for the legend items (trying out "Hind")
- Styled the new (awesome!) About modal a bit
<img width="259" alt="screen shot 2018-12-10 at 10 42 00 pm" src="https://user-images.githubusercontent.com/5672295/49777068-df056200-fccc-11e8-9f81-fffa77f41388.png">
<img width="522" alt="screen shot 2018-12-10 at 10 42 09 pm" src="https://user-images.githubusercontent.com/5672295/49777069-df056200-fccc-11e8-989e-7e9f54eae062.png">

pcaisse reviewed (approved):
:tada: :100: :+1: :fireworks:
